tests/core/test_jsonschema_helpers.py

```diff
@@ -434,6 +434,50 @@ def test_inbuilt_type(json_type: JSONTypeHelper, expected_json_schema: dict):
             },
             {is_integer_type},
         ),
+        (
+            Property(
```

Comment:
Would be nice to add this example to "Usage example" to have it in docs

Reply:
@TyShkan Thanks for the suggestion! I've updated the usage example: https://meltano-sdk--1603.org.readthedocs.build/en/1603/typing.html#usage-example